packages/terra-form-select/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Fixed Dropdown closing issue on Hold and Scroll in IE
+=======
```

Comment:
These === may be a result of resolving conflicts. They are appearing in the generated readme and should be removed.

![image](https://user-images.githubusercontent.com/6720991/50918996-541ca200-1407-11e9-8747-a13ba6414b2c.png)

Reply:
removed.